Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM alexcheng/magento2
+
+WORKDIR /var/www/html/ 
+
+COPY ./setup_magento /usr/local/bin/setup_magento
+
+RUN chmod +x /usr/local/bin/setup_magento
+
+ARG magento_username
+
+ARG magento_password
+
+COPY ./auth.json var/composer_home/auth.json
+
+RUN sed -i -e "s/user_placeholder/$magento_username/g" var/composer_home/auth.json && sed -i -e "s/pass_placeholder/$magento_password/g" var/composer_home/auth.json
+
+RUN chown -R www-data:www-data /var/www/html/var/composer_home
+
+USER www-data
```

Comment:
Can we document why we need to change to the www-data user? Maybe with a comment so other people can have context.

Reply:
Noted. I would add the comments